lib/marked.js

```diff
@@ -8,35 +8,54 @@
 'use strict';
 
 /**
- * Block-Level Grammar
+ * Markdown block-level element rules
+ *
+ * QUESTION: What is the different between heading the lheading?
```

Comment:
- heading is `# Heading` and `lheading

Reply:
Also this type of question should not be in the source code.

Reply:
@styfle: Think that's the crux of the documentation issue. Where do we put it. Initially I though describing the code in the contributing made the most sense because an outsider could read it before diving into the code. It was recommended to put it in the source code.

When I put on my "new guy" hat, these are the questions I'm asking when I look at the code. The question itself shouldn't remain in the source code once it's answered.

Reply:
> It was recommended to put it in the source code.